### PR TITLE
Improved clarity & structure of documentation in the installation section; fixed formatting

### PR DESCRIPTION
**Pull request by ahmvdev**
## PR Description

Hi, I've made some minor improvements in the documentation, particularly in the installation section. Here are the changes I have implemented: 

- Updated terminology from 'Python environment' to 'Python virtual environment' for clarity and accuracy.
- Added code formatting to 'conda' and 'conda-forge' for improved readability and consistency.
- Updated section title from 'Installing miniforge' to 'Installing miniforge (and conda)' to reflect the inclusion of conda in the installation process.
- Clarified the sentence by specifying that 'Miniforge' will install 'conda' for better context and understanding.
- "Revised the instruction for clarity: 'For Linux & Mac, run the downloaded script above using the following command:'.
- Updated the instruction to include restarting the system if changes don't take effect for better clarity.
- Updated terminology from 'environment' to 'virtual environment' in multiple instances for clarity and consistency.
- Added a command to verify the installation of 'sunpy' using 'conda list sunpy' for improved user guidance.

Thank you for reviewing my changes! I hope these improvements enhance the documentation. I'm open to any feedback or suggestions :)

### CODE REVIEW

File: docs/tutorial/installation.rst [hunk 1]
```diff
@@ -11,21 +11,21 @@ Installing Python
...
-Instead, we recommend that you use miniforge which is a minimal installer that setups conda with the conda-forge channel, which is free to use for everyone.
+Instead, we recommend that you use miniforge which is a minimal installer that setups ``conda`` with the ``conda-forge`` channel, which is free to use for everyone.
 If you are using miniforge, you can skip the next section
 
 .. _sunpy-tutorial-installing-miniforge:
 
-Installing miniforge
---------------------
+Installing miniforge (and conda)
+================
```
Comment (nabobalis) [inline docs/tutorial/installation.rst:25]: The under title has to match the length of the title

**Review by nabobalis — APPROVED** [review]

File: docs/tutorial/installation.rst (hunk 1 above)
Comment (ahmvdev) [reply docs/tutorial/installation.rst]: Thanks for the feedback! I have updated the subtitle length to match the title as requested please let me know if there's anything else that needs adjustment

**Comment by nabobalis** [comment]
Thank you @ahmvdev

**Comment by ahmvdev** [comment]
> Thank you @ahmvdev

Thanks for the merge! Looking forward to contributing more

**Pull request merged**